pyemma/coordinates/__init__.py

```diff
@@ -53,6 +34,27 @@
    cluster_uniform_time
    assign_to_centers
 
+Classes
+=======
+**Coordinate classes** encapsulating complex functionality. You don't need to construct these classes yourself, as this
+is done by the user API functions above. Find here a documentation how to extract features from them.
+
+.. autosummary::
+   :toctree: generated/
+
+   pipelines.Pipeline
+   transform.PCA
+   transform.TICA
 
 """
 from .api import *
+
+from .pipelines import Pipeline
```

Comment:
yes - will do. Still trying to find the issue, why none of the class
doc-strings is being created.